lib/hushed/black_tie_bundle.rb

```diff
@@ -2,32 +2,30 @@
 
 module Hushed
   module BlackTieBundle
-    extend self
+    module_function
```

Comment:
There's a commit that provides wider context on it 0463799 it's squashed so it'a a bit buried but the starting point is that Rubocop favors it.